R/write.reportProject.R

```diff
@@ -67,6 +67,11 @@ write.reportProject <- function(mif, mapping,
     }
    } else {
     stop("please provide either a path to a mif-file, a read in mif-file (in list-structure or as a magpie object)")
+   }
+
+  ### update logfile with current dataset name
```

Comment:
here one would have to check if `missing_log` is not NULL, but I also think this addition is not required. It is sufficient to print in case missing entries are found, see `RenameAndAggregate`.